### PR TITLE
React intro

### PR DESCRIPTION
**Pull request by matzimowski**

### CODE REVIEW

**Comment by matzimowski** [comment]
@kdzwinel take a look in free time, please.
all views are rendered by react.
data flow is totally fucked up, passing props to nested components sucks.
flux/redux needed for sure.

File: js/app.js [hunk 1]
```diff
@@ -1,148 +1,10 @@
...
+import React from 'react';
+import ReactDOM from 'react-dom';
+import SlideList from './components/slideList.js';
+
+(function start() {
+    ReactDOM.render(
+        <SlideList/>,
+        document.getElementById('app')
+    );
+})();
```
Comment (kdzwinel) [inline js/app.js]: I don't think we need a IIFE, this is a module, it will be "safe".

File: js/components/answer.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import VotesRepository from '../votes-repository';
+
+const Answer = React.createClass({
```
Comment (kdzwinel) [inline js/components/answer.js]: let's make it sexy with es6:

https://facebook.github.io/react/docs/reusable-components.html#es6-classes
> class HelloMessage extends React.Component {
>   render() {
>     return <div>Hello {this.props.name}</div>;
>   }
> }
> ReactDOM.render(<HelloMessage name="Sebastian" />, mountNode);

File: package.json [hunk 3]
```diff
@@ -18,12 +18,16 @@
...
+    "babel-preset-react": "^6.3.13",
     "babelify": "^7.2.0",
     "browserify": "^13.0.0",
     "gulp": "^3.9.0",
     "gulp-autoprefixer": "^3.1.0",
+    "gulp-react": "^3.1.0",
     "gulp-sass": "^2.1.1",
     "gulp-sourcemaps": "^1.6.0",
+    "react": "^0.14.7",
+    "react-dom": "^0.14.7",
```
Comment (kdzwinel) [inline package.json]: these two are dependencies, not devDependencies, right?

**Comment by kdzwinel** [comment]
fix :wrench: and merge!

File: package.json (hunk 3 above)
Comment (matzimowski) [inline package.json]: aight

**Comment by matzimowski** [comment]
@kdzwinel thanks for review!
it's just updated.

File: js/components/answer.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import VotesRepository from '../votes-repository';
+
+class Answer extends React.Component {
```
Comment (kdzwinel) [inline js/components/answer.js]: :heart:

**Comment by kdzwinel** [comment]
Merge merge babe

**Pull request merged**